appendices.tex

```diff
@@ -370,6 +370,152 @@ \chapter{Listing of application roles}
+
+\subsubsection{'environment' section}
+\label{subsubsec-global-config-ledgersmb-conf-environment}
+
+\begin{description}[style=nextline]
+	\item[PATH] @@@TODO
+\end{description}
+
+
+\subsubsection{'paths' section}
+\label{subsubsec-global-config-ledgersmb-conf-paths}
+
+\begin{description}[style=nextline]
+	\item [spool] @@@TODO
+	\item [userspath] @@@TODO
+	\item [templates] @@@TODO
+	\item [images] @@@TODO
+	\item [memberfile] @@@TODO
+	\item [localepath] @@@TODO
+\end{description}
+
+\subsubsection{'programs' section}
+\label{subsubsec-global-config-ledgersmb-conf-programs}
+
+\begin{description}[style=nextline]
+	\item [gzip] @@@TODO
+\end{description}
+
+
+\subsubsection{'mail' section}
+\label{subsubsec-global-config-ledgersmb-conf-mail}
+
+\begin{description}[style=nextline]
+	\item [sendmail] @@@TODO
+	\item [smtphost] @@@TODO
+	\item [smtptimeout] @@@TODO
+	\item [backup\_email\_from] @@@TODO
+\end{description}
+
+\subsubsection{'printers' section}
+\label{subsubsec-global-config-ledgersmb-conf-printers}
+
+This section contains a list of printers and the commands
+to be executed in order for the output to be sent to the given printer with
+the document to be printed fed to the program through standard input.
+The example below shows how to send standard input data to a printer called ``laser''
+when selecting the item ``Laser'' in the LedgerSMB printer selection drop down.
+
+\texttt{
+	Laser    = lpr -Plaser
+}
+
+
+\subsubsection{'database' section}
+\label{subsubsec-global-config-ledgersmb-conf-database}
+
+\begin{description}[style=nextline]
+	\item [host] Name of the host to connect to. See the documentation of the {\tt -h} command line option at 
+	\url{http://www.postgresql.org/docs/9.2/static/app-psql.html}
+	for more information (documentation unchanged since before 8.3, so applies to older versions as well)
+	\item [port] Port number to be used to connect to. See the documentation of the {\tt -p} option at the
+	same URL as the previous item
+	\item [default\_db] Database to connect to when the ``Company'' entry field in the login screen is left blank
+	\item [db\_namespace] The name space the company resides in; expert setting -- should not be used (default is
+	``public'')
+	\item [contrib\_dir] Path of the directory in which the PostgreSQL contrib modules (extensions) are
+	located; this setting is used by setup.pl when creating new companies or rebuilding (upgrading)
+	existing ones
+	\item [sslmode] Selects whether to use \gls{SSL} over TCP connections or not; can be ``require'', ``allow'',
+	``prefer'' or ``disable''
+\end{description}
+
+\subsubsection{log4perl\_config\_modules\_loglevel}
+\label{subsubsec-global-config-ledgersmb-conf-log4perl}
+
+@@@ This is a list of \textless module.name \textgreater = \textless level \textgreater
+
+This is an advanced setting not typically touched.
+
+
+\subsection{pos.conf.pl}
+\label{subsec-global-config-pos-conf}
+
+
+@@@ no idea what should go in here. to be investigated.

```

Comment:
The POS has been removed since a long time. The template content was there in 1.3: https://github.com/ledgersmb/LedgerSMB/blob/1.3/pos.conf.pl.template 

The POS was apparently removed at 1.4, because the POS code was there in 1.3 too (never noticed!) : https://github.com/ledgersmb/LedgerSMB/blob/1.3/bin/pos.pl

Reply:
OK, I'll remove it.

Reply:
Done.